tests/foreman/ui/test_leapp_client.py

```diff
@@ -71,6 +71,9 @@ def test_leapp_preupgrade_report(
         session.jobinvocation.wait_job_invocation_state(
             entity_name='Upgradeability check for rhel host', host_name=hostname
         )
+        session.jobinvocation.leapp_fix_inhibitor(
+            entity_name='Upgradeability check for rhel host', host_name=hostname
+        )
```

Comment:
what if there are multiple inhibitors to fix, how would you handle it here? 
and is this expected to have same entity_name as pre-upgrade check for fixing inhibitors?